appserver/monitoring-console/webapp/src/main/webapp/js/mc-view-components.js

```diff
@@ -78,20 +81,25 @@ MonitoringConsole.View.Components = (function() {
       function createTable(model) {
         let table = $('<table />', { id: model.id });
         if (model.caption)
-          table.append(createHeaderRow({ label: model.caption }));
+          table.append(createHeaderRow({ label: model.caption, description: model.description }));
         return table;
       }
 
       function createRow(model, inputs) {
         let components = $.isFunction(inputs) ? inputs() : inputs;
-        if (typeof components === 'string') {
+        if (typeof components === 'string')
```

Comment:
I would vote for using curly braces - do we have some standard for javascript here?

Reply:
I am not aware of some so I used a style that I think fits the language. If I would force curlies everywhere code gets 2 times as long for no good reason. Its just a lot of ifs in JS because you have to check all the possible craziness that callers can task functions with and screen size is finite - I rather have functions on screen as a whole then curlies on every if and for statement.